### ISSUE TITLE
UnicodeEncodeError on Windows while reading from directory with special characters

### ISSUE THREAD

**Issue by theruleof4** [digitalec/deemon#10]
After the new update,  seems some characters are not being reading correctly:

```
Message: 'Now monitoring ⣎⡇ꉺლ༽இ•̛)ྀ◞ ༎ຶ ༽ৣৢ؞ৢ؞ؖ ꉺლ'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "c:\users\valbe\appdata\local\programs\python\python37\lib\logging\__init__.py", line 1028, in emit
    stream.write(msg + self.terminator)
  File "c:\users\valbe\appdata\local\programs\python\python37\lib\encodings\cp1252.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
UnicodeEncodeError: 'charmap' codec can't encode characters in position 63-66: character maps to <undefined>
Call stack:
  File "c:\users\valbe\appdata\local\programs\python\python37\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\users\valbe\appdata\local\programs\python\python37\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\valbe\AppData\Local\Programs\Python\Python37\Scripts\deemon.exe\__main__.py", line 7, in <module>
    sys.exit(main())
  File "c:\users\valbe\appdata\local\programs\python\python37\lib\site-packages\deemon\__main__.py", line 4, in main
    cli.run()
  File "c:\users\valbe\appdata\local\programs\python\python37\lib\site-packages\click\core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "c:\users\valbe\appdata\local\programs\python\python37\lib\site-packages\click\core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "c:\users\valbe\appdata\local\programs\python\python37\lib\site-packages\click\core.py", line 1668, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "c:\users\valbe\appdata\local\programs\python\python37\lib\site-packages\click\core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "c:\users\valbe\appdata\local\programs\python\python37\lib\site-packages\click\core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "c:\users\valbe\appdata\local\programs\python\python37\lib\site-packages\deemon\cli.py", line 145, in import_cmd
    batch.import_artists(path)
  File "c:\users\valbe\appdata\local\programs\python\python37\lib\site-packages\deemon\app\batch.py", line 40, in import_artists
    ma.start_monitoring(silent=True)
  File "c:\users\valbe\appdata\local\programs\python\python37\lib\site-packages\deemon\app\monitor.py", line 58, in start_monitoring
    logger.debug("Now monitoring " + self.artist)
Message: 'Now monitoring ２８１４'
Arguments: ()
```

**Comment by digitalec** [comment]
Can you provide some more info?

What OS?

What is the full command you're running?

What language/keyboard layout are you using?

**Comment by theruleof4** [comment]
> Can you provide some more info?
> 
> What OS?
> 
> What is the full command you're running?
> 
> What language/keyboard layout are you using?

Oh sorry, my mistake:

Win 10,
Command: deemon import
Keyboard ABNT 2

I guess is a problem with the artist character, because this error happened with these artists: ２８１４, ⣎⡇ꉺლ༽இ•̛)ྀ◞ ༎ຶ ༽ৣৢ؞ৢ؞ؖ ꉺლ', тпсб, which uses some specials characters

**Comment by digitalec** [comment]
Would you be able to attach a zip file containing the text file you're using to import so I can test?

**Comment by theruleof4** [comment]
> Would you be able to attach a zip file containing the text file you're using to import so I can test?

What do you mean? I'm importing from a path

**Comment by digitalec** [comment]
Sorry, I meant on here. Can you zip the text file you're using and attach it on your next comment here on this issue.

**Comment by theruleof4** [comment]
I can not understand, I'm using the command `deemon import C:\music`, so it's grabbing from a folder and not a list from a text file.
The other artists worked fine, but these with special characters don't

**Comment by digitalec** [comment]
Ok, I assumed you were using a text file to import.

Are the characters that I see in my screenshot correct? If not, can you send me a screenshot of the error?

![image](https://user-images.githubusercontent.com/31838395/120943558-aad45600-c6fd-11eb-8281-3e88270ffb2b.jpeg)

**Comment by theruleof4** [comment]
I see, check the screenshot:

The characters are correct, these artists exists on deezer:

https://www.deezer.com/search/%E2%A3%8E%E2%A1%87%EA%89%BA%E1%83%9A%E0%BC%BD%E0%AE%87%E2%80%A2%CC%9B)%E0%BE%80%E2%97%9E%20%E0%BC%8E%E0%BA%B6%20%E0%BC%BD%E0%A7%A3%E0%A7%A2%D8%9E%E0%A7%A2%D8%9E%D8%96%20%EA%89%BA%E1%83%9A

https://www.deezer.com/search/%D1%82%D0%BF%D1%81%D0%B1

I've searched with the same name and was found

![image](https://user-images.githubusercontent.com/62116034/120943888-03105580-c708-11eb-9b5f-9af8c9ac099b.png)

**Comment by digitalec** [comment]
I just pushed v1.0.1-beta1 to pip to try and fix this issue. Can you try it out?

`pip install deemon==1.0.1b1`

**Comment by digitalec** [comment]
Bug has been fixed and published in v1.0.1

**Comment by theruleof4** [comment]
Now, worked well. Thanks!
